source/sonata_report.rst

```diff
@@ -70,28 +70,180 @@ This field is not implemented.
+By targeting these processes instead of the synapses themselves, the report can efficiently track only plastic synapses while maintaining direct access to weight changes without impacting synaptic transmission performance.
+
+For synapse reports, you need to specify ``"type": "synapse"`` in the configuration. Note that ``"sections": "all"`` must be included to capture synapses on all compartments; otherwise, only somatic synapses will be reported::
+
+    "reports": {
+        "synapse_weights": {
+            "cells": "Column",
+            "type": "synapse",
+            "variable_name": "StdpWADoublet.weight",
+            "sections": "all",
+            "unit": "nS",
+            "dt": 0.1,
+            "start_time": 0,
+            "end_time": 1000
+        },
+        "exc_conductances": {
+            "cells": "AllCompartments",
+            "type": "synapse",
+            "variable_name": "ProbAMPANMDA_EMS.g",
+            "sections": "all",
+            "unit": "nA",
+            "dt": 0.1,
+            "start_time": 0.025,
+            "end_time": 100
+        }
+    }
 
 LFP report
 ^^^^^^^^^^
 
-In this case, the element_ids are the electrode ids described in the electrodes_file :ref:`here <sonata_tech>`.
+A specialized report for recording Local Field Potentials (LFPs) that captures the aggregate electrical activity of neural populations. The report structure uses:
+
+* ``element_ids``: Electrode identifiers defined in the electrodes_file (:ref:`sonata_tech`)
+* ``node_ids``: Identifiers of the contributing cells
+
+LFP reports require pre-calculated weight file (or electrodes file) that define the contribution of each compartment to the LFP signal at each electrode position.
+
+For lfp reports, you need to specify ``"type": "lfp"`` and ``"target_simulator": "CORENEURON"`` in the configuration::
+
+    "target_simulator": "CORENEURON",
+    "run": {
+        "tstart": 0,
+        ...
+        "electrodes_file": "/path/to/electrodes_file.h5"  # Contains electrode positions
+    }
+
+    "reports": {
+        "lfp_report": {
+            "type": "lfp",
+            "cells": "Mosaic",
+            "variable_name": "v",
+            "dt": 0.1,
+            "start_time": 0.0,
+            "end_time": 40.0
+        }
+    }
+
+Important Notes:
+
+* Electrode positions must be defined before simulation
+* Uses same HDF5 structure as compartment reports
+* element_ids correspond to electrode positions
+* Commonly used for analyzing population-level activity
+
+For more detailed information about LFP reports, please refer to the `Online LFP Calculation Documentation <https://github.com/BlueBrain/neurodamus/blob/main/docs/online-lfp.rst#online-lfp-calculation-documentation>`_.
 
 Bloodflow report
 ^^^^^^^^^^^^^^^^
```

Comment:
We support bloodflow report? What is the report type ?

Reply:
This was not added by me. Those are not done in the conventional way. 
tbh I dont know how is this generated
@cattabiani probably knows?

Reply:
I did not add this. Maybe Dan? I cannot @

Reply:
maybe it comes from vascCouplingB?

Reply:
This type of report is created by the bloodflow solver. 
https://github.com/BlueBrain/AstroVascPy/blob/d40a16b78bf5c4b1f02b8d8f6f8cb30b316b9dd9/astrovascpy/report_writer.py#L19
It is clear now, we keep it like this.